tests/unit/ui/test_console.py

```diff
@@ -15,6 +17,34 @@ def test_write(capsys: CaptureFixture[str]):
     assert captured.err == f"{message}\n"
 
 
+@pytest.mark.parametrize("isatty", [True, False])
```

Comment:
Let's use parametrize for the `expected_out` as well.
```suggestion
@pytest.mark.parametrize("isatty, expected", [
    (True, textwrap.dedent("""\
         {"hello": "world"}
    """)),
    (False, '{"hello": "world"}\n'),
])
```

Reply:
ok